### PR TITLE
FIX: Allow setting nonstandard spaces for parser

### PR DESCRIPTION
**Pull request by oesteban**
Addresses [this problem](https://app.circleci.com/jobs/github/poldracklab/fmriprep/10357) - ref.: poldracklab/fmriprep#1827.

### CODE REVIEW

File: smriprep/cli/utils.py [hunk 1]
```diff
@@ -27,16 +29,29 @@ def _template(inlist):
...
+
     if template not in _TF_TEMPLATES:
         raise ValueError("""\
 Template identifier "{}" not found. Please, make sure TemplateFlow is \
 correctly installed and contains the given template identifiers.""".format(template))
 
     return template, spec
+
+
+def set_nonstandard_spaces(inlist):
```
Comment (effigies) [inline smriprep/cli/utils.py]: This function doesn't seem to be called anywhere.

Comment (oesteban) [reply smriprep/cli/utils.py]: https://github.com/adelavega/fmriprep/commit/c7cddba027946cc8e9c1d524a98b2b855a032adc#diff-b93ab0122f4b2ac6738b896651cac6ccR53

Comment (effigies) [reply smriprep/cli/utils.py]: Oh, I see...

Is there another way to do this? It feels very messy to fiddle with global state. What if we just passed a list of valid nonstandard spaces to `output_space`?

**Comment by oesteban** [comment]
How would you pass it to ParseTemplate?

File: smriprep/cli/utils.py (hunk 1 above)
Comment (oesteban) [reply smriprep/cli/utils.py]: How would you pass it to `ParseTemplates`?

Comment (effigies) [reply smriprep/cli/utils.py]: IDK... I'm not really aware of all this machinery. I'll look tomorrow to see if I can figure out what we're doing with CLI args these days and if there's something cleaner. If not, I guess do what you have to.

Comment (oesteban) [reply smriprep/cli/utils.py]: For sure I can rework the action class... but I honestly took the lazy way for this.

Comment (oesteban) [reply smriprep/cli/utils.py]: @effigies, more acceptable now?

**Pull request merged**